### DISCUSSION TITLE
Saleor 3.1 | Changelog

### DISCUSSION THREAD

**Discussion by karolkielecki** [saleor/saleor#9265] [Show and tell]
Welcome to Saleor 3.1! 🚀 

What's new this time? Gift Cards, Click & Collet, improved extensibility, and more. Check the full changelog below. 
https://github.com/saleor/saleor/releases/tag/3.1.0

Please bear in mind that some of the new features of 3.1 are in a preview state. A blog post summarizing the most important features will be published soon. Sign up for our [newsletter](https://bit.ly/Saleor_Newsletter)!

---
[Twitter](https://twitter.com/getsaleor/status/1498269559194886147)

**Comment by fijolekProjects** [comment]
Thank you for another release! :gift: 

Currently we're on Saleor `3.0` and it's great so far. We've forked it and made custom changes to align it with our business needs and we've been keeping our fork updated with `3.0` for some time which worked great. However having Saleor fork may not be recommended way of extending Saleor.

I see that `3.0` and `3.1` were developed separately and simultaneously which means that upgrade from fork of 3.0 to 3.1 could be tricky, since `3.1` is not based on `3.0` in git. 
1. Is `3.0` branch currently in maintenance mode and no new futures will be added there?
2. What way of keeping custom Saleor fork updated with stable release do you recommend apart from Saleor Apps and Plugins?

Thank you

**Reply by karolkielecki** [reply]
Hey @fijolekProjects, thanks for your support! 
Re your questions:
1. We're currently focusing on 3.1+. There won't be any new features added to 3.0. We've just released [3.1.1 ](https://github.com/saleor/saleor/releases/tag/3.1.1).
2. We don't have any guide on how to update a custom Saleor fork to the latest stable releases. Let me ask the team. 

Cheers,
